inspire/modules/formatter/views.py

```diff
@@ -84,3 +88,36 @@ def get_latex_file(recid, latex_format):
             )
         }
     )
+
+
+@blueprint.route("/export-as/<path:export_format>/<path:ids>",
+                 methods=['GET', 'POST'])
+def download_all_latex(ids, export_format):
+    out = []
+    ids_list = ids.split(',')
+    for element in ids_list:
+        record = get_record(element)
+        if export_format == 'bibtex':
+            results = Bibtex(record).format()
+        elif export_format in ('latex_eu', 'latex_us'):
+            results = Latex(record, export_format).format()
+        elif export_format == 'cv_latex':
+            results = Cv_latex(record).format()
+        elif export_format in ('cv_latex_html', 'cv_latex_text'):
+            results = Cv_latex_html_text(record, export_format).format()
+
+        generator = (cell for row in results
+                     for cell in row)
+        out.append(generator)
+    output = itertools.chain()
+    for gen in out:
+        output = itertools.chain(output, gen)
+
+    return Response(
+        output,
+        mimetype="text/plain",
+        headers={
+            "Content-Disposition": "attachment;filename=allLatex_{0}.tex".format(
```

Comment:
if the output format is bibtex we have the_file.bibtex, if latex the_file.tex. If it is one of the others should I save it as txt? any ideas?